content/1.7/back-office/orders/orders/view-shopping-cart.md

```diff
@@ -0,0 +1,36 @@
+---
+title: View Shopping Cart listing
+weight: 1
+---
+
+# SELL > Orders > Shopping Carts > View
+
+[View Shopping Cart](static/img/orders-shopping-carts-view-shopping-cart.png)
+
+This dashboard is displaying the information about the Shopping Cart initiator - the customer, visitor or guest visitor who generated the Cart in Prestashop. It can be accessed by selecting the Shopping Cart row and clicking on View CTA button on the right of the row.
```

Comment:
```suggestion
This dashboard is displaying the information about the Shopping Cart initiator - the customer, visitor or guest visitor who generated the Cart. It can be accessed by selecting the Shopping Cart row and clicking on View button on the right of the row.
```